dvc/api.py

```diff
@@ -63,7 +70,7 @@ def _open(path, repo=None, rev=None, remote=None, mode="r", encoding=None):
 
 
 def read(path, repo=None, rev=None, remote=None, mode="r", encoding=None):
-    """Returns the contents of a tracked file."""
+    """Returns the contents of a tracked file (by DVC or Git). Wraps open()"""
```

Comment:
`Wraps open()` is not important for consumers here.

Reply:
OK, then I need to copy paste the rest of the docstring from open() – about defaults, etc.

Reply:
We should probably drop `mode` though, it doesn't make sense for read. Set 
```python
mode = "rb" if encoding is None else "r"
```

Reply:
OK, updated in f29e5ecb.

> We should probably drop mode

You mean the actual fn parameter @Suor? OK, up to you (separate PR I assume) but it's not mentioned in the docstring anyway 🙂

Reply:
p.s. Resolving this one.